v2/delay/delay.go

```diff
@@ -162,14 +168,18 @@ func fileKey(file string) (string, error) {
 func Func(key string, i interface{}) *Function {
 	f := &Function{fv: reflect.ValueOf(i)}
 
-	// Derive unique, somewhat stable key for this func.
+	// Derive unique, somewhat stable key for this func, unless users specify a unique key.
 	_, file, _, _ := runtime.Caller(1)
-	fk, err := fileKey(file)
-	if err != nil {
-		// Not fatal, but log the error
-		stdlog.Printf("delay: %v", err)
-	}
-	f.key = fk + ":" + key
+  if !strings.HasPrefix(key, "unique:") {
```

Comment:
Please create an exported const for this, and update your example above to use the const.